pystoned/CNLS.py

```diff
@@ -197,8 +194,7 @@ def afriat_rule(model, i, h):
 
                 return afriat_rule
             elif self.rts == RTS_CRS:
-                # TODO(warning handling): replace with model requested not exist
-                return False
+                raise Exception("The Additive CNLS with constant return to scale does not exist")

```

Comment:
Please add CRS specifications to all additive CNLS models

Reply:
The CRS with additive model has been added to CNLS model. The other related model will be modified in another fix commit.